### DISCUSSION TITLE
Disable default tags and context

### DISCUSSION THREAD

**Discussion by DrummerB** [getsentry/sentry-unity#880] [Q&A] [answered]
Is there a way to completely disable the SDK-defined default context and tags that are attached to events? I would like to define the context and tags myself.

**Comment by bitsandfoxes** [comment, marked as answer]
Most of that happens inside of `ScopeIntegration` and there is currently no mechanism to remove it. Is there anything in particular you want to get rid of?

**Reply by DrummerB** [reply]
As a workaround, I was able to modify the default context in an ISentryEventProcessor. Example:
```cs
public class SentryEventProcessor : ISentryEventProcessor
{
	public SentryEvent Process(SentryEvent sentryEvent)
	{
		if (sentryEvent.Contexts.TryRemove("ThreadPool Info", out object threadInfo))
			sentryEvent.Contexts["thread pool info"] = threadInfo;
		// ...
	}
}
```
I hope this doesn't mess up any internal state in the SDK.

My reason for this is a bit nit-picky. The SDK supplies some context with PascalCase keys and others with camelCase. The web UI uses case-sensitive sorting but displays them in ALLCAPS. This makes the order appear random and a bit unintuitive to navigate.

**Reply by bitsandfoxes** [reply]
We've had this discussion with the team and the current behaviour is: 
- Unity specific contexts - ordered alphabetically
- SKD "agnostic" context (that can be found in any project) - ordered alphabetically

But I agree, it does not end up being super intuitive when you're looking for something.
We'll try to clean up the different casings.

**Reply by DrummerB** [reply]
Is this sorting implemented on the client-side or does the server or frontend do it? From what I could tell, the order of the keys within the SentryEvent.Contexts dictionary does not matter (and probably should now, as it is technically undefined) and the contexts are always sorted alphabetically, except for the optional User context, which will be at the top, and the Assemblies and SDK contexts, which are at the bottom.

This is probably out-of-scope for sentry-unity, but I think the order should be up to the client (with some callbacks to override it) and determined by the order in the transmitted JSON. And a search field, plus collapsed/collapsible context sections, would help a lot in navigating the available info.
